Cargo.toml

```diff
@@ -18,5 +18,4 @@ lazy_static = "1.4.0"
 regex = "1.4.3"
 phf = { version = "0.8", features = ["macros"] }
 strum = { version = "0.20", features = ["derive"] }
```

Comment:
I'm thinking of adding a `nightly` feature to be able to release the library sooner 🤔 what do you think?

Reply:
Firstly, our development speed is not high. Secondly, this library does not have many users yet, so I think it is not necessary

Reply:
So you mean we can release version 0.1.0?